examples/google/example.go

```diff
@@ -1,176 +1,35 @@
 package main
 
 import (
-	"crypto"
-	"crypto/rand"
-	"crypto/sha256"
-	"crypto/x509"
-	"encoding/base64"
-	"encoding/hex"
-	"encoding/pem"
+	"bytes"
+	"encoding/json"
 	"fmt"
 	"os"
 
-	"github.com/sirupsen/logrus"
+	"github.com/lestrrat-go/jwx/v2/jwa"
+	"github.com/lestrrat-go/jwx/v2/jwk"
 	"github.com/zitadel/oidc/v2/pkg/oidc"
 
 	"github.com/openpubkey/openpubkey/parties"
-	"github.com/openpubkey/openpubkey/pktoken"
+	"github.com/openpubkey/openpubkey/util"
 )
 
 // TODO: Create nice golang services rather than just using this handler nonsense
 
 type ReceiveIDTHandler func(tokens *oidc.Tokens[*oidc.IDTokenClaims])
 
-func GoogleSign() {
-	signer, err := pktoken.LoadFromFile(fpClientCfg, "ES256", false, nil)
-	if err != nil {
-		logrus.Fatalf("Error loading client state: %s", err.Error())
-		return
-	}
-
-	client := &parties.OpkClient{
-		Signer: signer,
-		Op: &parties.GoogleOp{
-			ClientID:     clientID,
-			ClientSecret: clientSecret,
-			Issuer:       issuer,
-			Scopes:       scopes,
-			RedirURIPort: redirURIPort,
-			CallbackPath: callbackPath,
-			RedirectURI:  redirectURI,
-		},
-	}
-
-	msg, err := os.ReadFile("hunter.txt")
-	if err != nil {
-		panic(err)
-	}
-
-	msgHash := sha256.New()
-	_, err = msgHash.Write(msg)
-	if err != nil {
-		panic(err)
-	}
-	msgHashSum := msgHash.Sum(nil)
-	hashhex := hex.EncodeToString(msgHashSum)
-
-	rawSigma, err := client.Signer.Pksk.Sign(rand.Reader, msgHashSum, crypto.SHA256)
-	if err != nil {
-		logrus.Fatalf("Error signing: %s", err.Error())
-		return
-	}
-
-	err = os.WriteFile("sigma.sig", rawSigma, 0644)
-
-	sigma := base64.StdEncoding.EncodeToString(rawSigma)
-	if err != nil {
-		panic(err)
-	}
-	cert, err := os.ReadFile("pkt.cert")
-	if err != nil {
-		panic(err)
-	}
-
-	certstr := make([]byte, base64.StdEncoding.EncodedLen(len(cert)))
-	base64.StdEncoding.Encode(certstr, cert)
-
-	fmt.Println("Praise Sigma: " + string(sigma))
-	fmt.Println("Hash: " + string(hashhex))
-	fmt.Println("Cert: " + string(certstr))
-}
-
-func GoogleCert() {
-
-	signer, err := pktoken.LoadFromFile(fpClientCfg, "ES256", false, nil)
-	if err != nil {
-		fmt.Printf("Error loading client state: %s", err.Error())
-		return
-	}
-
-	client := &parties.OpkClient{
-		Op: &parties.GoogleOp{
-			ClientID:     clientID,
-			ClientSecret: clientSecret,
-			Issuer:       issuer,
-			Scopes:       scopes,
-			RedirURIPort: redirURIPort,
-			CallbackPath: callbackPath,
-			RedirectURI:  redirectURI,
-		},
-		Signer: signer,
-	}
-
-	certBytes, err := client.RequestCert()
-	if err != nil {
-		fmt.Printf("Error signing: %s", err.Error())
-		return
-	}
-	fmt.Println("Cert received: " + string(certBytes))
-
-	block, _ := pem.Decode(certBytes)
-	cert, err := x509.ParseCertificate(block.Bytes)
-	if err != nil {
-		fmt.Printf("Error failed to parse cert: %s", err.Error())
-		return
-	}
```

Comment:
This code is useful, don't delete. It will be changed and updated soon

Reply:
This is pending getting the functionality fixed on main

Reply:
adressed!